src/transformers/models/granitemoehybrid/modular_granitemoehybrid.py

```diff
@@ -51,8 +51,8 @@ def __init__(self, config: GraniteMoeHybridConfig, layer_idx: int):
 
 
 class GraniteMoeHybridRMSNormGated(BambaRMSNormGated):
-    def __init__(self, hidden_size, eps=1e-6):
-        super().__init__(hidden_size, eps)
+    def __init__(self, hidden_size, group_size, eps=1e-6):
+        super().__init__(hidden_size, group_size, eps)
```

Comment:
```suggestion
    pass
```
Weird that this even used the init here, we shouldn't need to anything on the modular side

Reply:
Same for the other modular codes